dev/io.openliberty.microprofile.telemetry.internal.monitor_fat/fat/src/io/openliberty/microprofile/telemetry/internal/monitor_fat/FATSuite.java

```diff
@@ -25,14 +25,12 @@
 				SessionMetricsTest.class,
 				ConnectionPoolMetricsTest.class})
 public class FATSuite extends TestContainerSuite {
-	
-    public static RepeatTests testRepeatMPTel20(String serverName) {
-        return RepeatTests.with(FeatureReplacementAction.EE11_FEATURES())
-                        .andWith(FeatureReplacementAction.EE10_FEATURES().fullFATOnly())
-                        .andWith(FeatureReplacementAction.EE9_FEATURES().fullFATOnly())
-                        .andWith(FeatureReplacementAction.EE8_FEATURES().fullFATOnly())
-                        .andWith(FeatureReplacementAction.EE7_FEATURES().fullFATOnly());
 
+    public static RepeatTests testRepeatMPTel20(String serverName) {
```

Comment:
I would recommend using the TelemetryActions / MicroProfileActions common classes for this.

Reply:
Can we really do that since we want mpTelemetry-2.0 for all of them?  No other MP features are used.

Reply:
Yes, MicroProfileActions does not add features which are not already in the server.xml... but as Andrew points out below, it does automatically make sure that something is always available to be run in LITE mode.